td-shim/src/bin/td-shim/memory.rs

```diff
@@ -198,6 +205,15 @@ impl<'a> Memory<'a> {
         regions
     }
 
+    fn support_unaccepted_type(resources: &[ResourceDescription]) -> bool {
```

Comment:
Please add detail comment on why we need this.

Reply:
Done